Sources/EmbraceCore/Internal/Embrace+Setup.swift

```diff
@@ -91,7 +91,7 @@ extension Embrace {
 
         do {
             let options = EmbraceUpload.Options(endpoints: uploadEndpoints, cache: cache, metadata: metadata)
-            let queue = DispatchQueue(label: "com.embrace.upload", qos: .background, attributes: .concurrent)
+            let queue = DispatchQueue(label: "com.embrace.upload", qos: .utility)
```

Comment:
Made the queue .utility. The `.background` queue might not run in certain circumstances like the app being in the background (yeah, I know it's weird) or other low power/memory situations.